### ISSUE TITLE
[Package Issue]: JohnMacFarlane.Pandoc

### ISSUE THREAD

**Issue by chrisnestrud** [microsoft/winget-pkgs#118528]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

Unable to install using --scope=user

### Steps to reproduce

winget install --scope=user JohnMacFarlane.Pandoc

### Actual behavior

Version 3.1.2 is installed.

### Expected behavior

Version 3.1.6.2 is installed; there is an entry for this version in the manifest but it is lacking the metadata for user scope.

### Environment

```raw
Windows Package Manager v1.5.2201
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.3324
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.20.2201.0

Winget Directories
-----------------------------------------------------------------------------------------------------------------------
Logs                               %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\Diag…
User Settings                      %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\sett…
Portable Links Directory (User)    %LOCALAPPDATA%\Microsoft\WinGet\Links
Portable Links Directory (Machine) C:\Program Files\WinGet\Links
Portable Package Root (User)       %LOCALAPPDATA%\Microsoft\WinGet\Packages
Portable Package Root              C:\Program Files\WinGet\Packages
Portable Package Root (x86)        C:\Program Files (x86)\WinGet\Packages

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale

Admin Setting                             State
--------------------------------------------------
LocalManifestFiles                        Enabled
BypassCertificatePinningForMicrosoftStore Disabled
InstallerHashOverride                     Disabled
LocalArchiveMalwareScanOverride           Disabled
```

### Screenshots and Logs

The JohnMacFarlane/Pandoc/3.1.6.2/JohnMacFarlane.Pandoc.installer.yaml file does not have an installer entry for user scope.

Under InstallerSwitches it needs:
Custom: ALLUSERS=2

**Comment by Trenly** [comment]
Close with reason: Resolved in newer version; PR #119502

**Comment by apaulfox** [comment]
The user scope install is broken in pandoc 3.6.2
